deploy/examples/create-external-cluster-resources.py

```diff
@@ -205,6 +207,44 @@ def gen_arg_parser(cls, args_to_parse=None):
             args_to_parse = sys.argv[1:]
         return argP.parse_args(args_to_parse)
 
+    def validate_rgw_metadata_ec_pool_name(self):
+        if self._arg_parser.rbd_metadata_ec_pool_name:
+            rbd_metadata_ec_pool_name = self._arg_parser.rbd_metadata_ec_pool_name
+            rbd_pool_name = self._arg_parser.rbd_data_pool_name
+
+            if rbd_pool_name == "" or rbd_metadata_ec_pool_name == "":
```

Comment:
Let's do it in 2 steps so the error message clearly indicates which flag is not set